### ISSUE TITLE
Version 0.21 install fail

### ISSUE THREAD

**Issue by slmg** [bpython/bpython#880]
Installing in a linux OS

pip version `20.3.3`

```sh
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/.venvs/bpython/bin/python /usr/local/bin/.venvs/bpython/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py build_wheel /tmp/tmp2q79061b
       cwd: /tmp/pip-install-9cqvozjs/cwcwidth
  Complete output (29 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.8
  creating build/lib.linux-x86_64-3.8/cwcwidth
  copying cwcwidth/__init__.py -> build/lib.linux-x86_64-3.8/cwcwidth
  warning: build_py: byte-compiling is disabled, skipping.
  
  running build_ext
  building 'cwcwidth._impl' extension
  creating build/temp.linux-x86_64-3.8
  creating build/temp.linux-x86_64-3.8/cwcwidth
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -D_XOPEN_SOURCE=600 -I/usr/local/bin/.venvs/bpython/include -I/usr/local/include/python3.8 -c cwcwidth/_impl.c -o build/temp.linux-x86_64-3.8/cwcwidth/_impl.o
  In file included from /usr/local/include/python3.8/Python.h:8,
                   from cwcwidth/_impl.c:28:
  /usr/local/include/python3.8/pyconfig.h:1612: warning: "_XOPEN_SOURCE" redefined
   #define _XOPEN_SOURCE 700
  
  <command-line>: note: this is the location of the previous definition
  In file included from /usr/lib/gcc/x86_64-linux-gnu/8/include-fixed/syslimits.h:7,
                   from /usr/lib/gcc/x86_64-linux-gnu/8/include-fixed/limits.h:34,
                   from /usr/local/include/python3.8/Python.h:11,
                   from cwcwidth/_impl.c:28:
  /usr/lib/gcc/x86_64-linux-gnu/8/include-fixed/limits.h:194:15: fatal error: limits.h: No such file or directory
   #include_next <limits.h>  /* recurse down to the real one */
                 ^~~~~~~~~~
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for cwcwidth
  Building wheel for curtsies (PEP 517): started
  Building wheel for curtsies (PEP 517): finished with status 'done'
  Created wheel for curtsies: filename=curtsies-0.3.5-py3-none-any.whl size=35190 sha256=876c749990b52fb84c250a633adc5d3b6d3bf71ffc0576828b85e58e34d87632
  Stored in directory: /tmp/pip-ephem-wheel-cache-rgpw5ry5/wheels/5e/0c/93/071e8cd9549a478fe062b9d3f3680c69d857ca75286dd30b47
Successfully built curtsies
Failed to build cwcwidth
ERROR: Could not build wheels for cwcwidth which use PEP 517 and cannot be installed directly
```

**Comment by sebastinas** [comment]
Let's move this to https://github.com/sebastinas/cwcwidth

**Comment by slmg** [comment]
Created https://github.com/sebastinas/cwcwidth/issues/2.

Workaround till cwcwidth is fixed: `pip install 'bpython<0.21'  'curtsies<0.3.4'`
